.eslintrc

```diff
@@ -35,6 +35,18 @@
     "es6": true
   },
 
+  "globals": {
```

Comment:
Let's get linting into the codebase after this PR merges sure :) It should be easy to add additional gulp tasks for linting the codebase and test directories separately.